tests/contrib/gunicorn/wsgi_mw_app.py

```diff
@@ -10,11 +29,28 @@
 )
 
 
+def aggressive_shutdown():
+    tracer.shutdown(timeout=1)
+    RemoteConfig.disable()
+    DynamicInstrumentation.disable()
+    bootstrap.profiler._scheduler.stop()
```

Comment:
this should get handled by `profiler.stop()`


https://github.com/DataDog/dd-trace-py/blob/6bf92f8fda1b7e5404a22320b96def4b536ee89c/ddtrace/profiling/profiler.py#L78

->

https://github.com/DataDog/dd-trace-py/blob/6bf92f8fda1b7e5404a22320b96def4b536ee89c/ddtrace/profiling/profiler.py#L286-L293

Reply:
For some reason this "aggressive shutdown" approach appears to be necessary under the test conditions I'm creating. Without it, some of the tests hang forever.